TypeScript.tmLanguage

```diff
@@ -3172,6 +3172,11 @@
           <key>2</key>
           <dict>
             <key>name</key>
+            <string>keyword.control.loop.ts</string>
```

Comment:
I tried that but the coloring looked odd in the editor, plus `await` in this context isn't control flow, but modifies the loop behavior.